source/renderer/app/config/analyticsConfig.ts

```diff
@@ -0,0 +1,14 @@
+import { Network } from '../../../common/types/environment.types';
+
+export const ANALYTICS_API_ENDPOINT = 'https://mazurek.matomo.cloud/matomo.php';
```

Comment:
TODO

Reply:
Is this intended for prod?

Reply:
No. We're waiting for Legal team. Once they give us a green light, then we can spin up a Matomo instance and update this endpoint. This PR is labelled as `DO NOT MERGE` because of that.